src/Microsoft.PowerShell.ConsoleHost/host/msh/CommandLineParameterParser.cs

```diff
@@ -185,7 +185,7 @@ internal class CommandLineParameterParser
             "file",
```

Comment:
@rjmholt, your last commit had 6 failures in `PowerShell-CI-macos`
(These are 5 of the failures)

ConsoleHost unit tests.-Login pwsh switch.Accepts the -l switch for -Login and behaves correctly
```powershell
Expected 3198520b-afb2-4339-b864-3c12ba3fa282, but got $null.
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Host/ConsoleHost.Tests.ps1: line 295
295:             $result | Should -Be $guid
```
ConsoleHost unit tests.-Login pwsh switch.Accepts the -L switch for -Login and behaves correctly
```powershell
Expected 3198520b-afb2-4339-b864-3c12ba3fa282, but got $null.
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Host/ConsoleHost.Tests.ps1: line 295
295:             $result | Should -Be $guid
```
ConsoleHost unit tests.-Login pwsh switch.Accepts the -login switch for -Login and behaves correctly
```powershell
Expected 3198520b-afb2-4339-b864-3c12ba3fa282, but got $null.
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Host/ConsoleHost.Tests.ps1: line 295
295:             $result | Should -Be $guid
```
ConsoleHost unit tests.-Login pwsh switch.Accepts the -Login switch for -Login and behaves correctly
```powershell
Expected 3198520b-afb2-4339-b864-3c12ba3fa282, but got $null.
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Host/ConsoleHost.Tests.ps1: line 295
295:             $result | Should -Be $guid
```
ConsoleHost unit tests.-Login pwsh switch.Accepts the -LOGIN switch for -Login and behaves correctly
```powershell
Expected 3198520b-afb2-4339-b864-3c12ba3fa282, but got $null.
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Host/ConsoleHost.Tests.ps1: line 295
295:             $result | Should -Be $guid
```